lib/system.nim

```diff
@@ -569,7 +569,12 @@ type
       trace: string
     else:
       trace: seq[StackTraceEntry]
-    raiseId: uint # set when exception is raised
+    when defined(nimBoostrapCsources0_19_0):
+      # see #10315, bootstrap with `nim cpp` from csources gave error:
+      # error: no member named 'raise_id' in 'Exception'
+      raiseId: uint # set when exception is raised
```

Comment:
shouldn't this be ``raise_id``? This makes no sense.

Reply:
right, it was and reverted it by mistake, ya, will update (as the approach you committed is slower and will impact CI, which is already dying in C++ mode)